get-started/part3.md

```diff
@@ -103,6 +103,19 @@ This `docker-compose.yml` file tells Docker to do the following:
 - Define the `webnet` network with the default settings (which is a
   load-balanced overlay network).
 
+Notice that we've set the Compose file version to version 3. This essentially
+makes it [swarm mode](/engine/swarm/index.md) compatible. We can make use of the
```

Comment:
I'm not sure about this explanation. I mean, it's kind of true, but it's not complete. Maybe that's ok in this context. Like setting the version to 3 actually uses the version of the Compose API which includes the swarm mode stuff, right? The version specifies the version of the Compose API you can use.